node/test/OpenT2TTests.ts

```diff
@@ -141,12 +142,29 @@ test("Logger with default parameters can be instantiated multiple times", async
 });
 
 test("Logger can be instantiated with custom file parameter", async t => {
-    let logger = new Logger(undefined, testLogFileName) ;
+    let logger = new Logger(undefined, testLogFileName);
     logger.info("Writing default level to default console + file.");
     logger.warn("writing warn level to default console + file.");
     t.is(logger.getConfiguredTransports().length, 2);
 });
 
+test("Logger with default parameters can log custom loggingMetadata parameter like a correlationVector", async t => {
```

Comment:
Fixed

---
In reply to: [105495418](https://github.com/openT2T/opent2t/pull/38#discussion_r105495418) [](ancestors = 105495418)